File: java/none/library.g8/src/main/g8/src/test/java/$package$/GreeterTest.java
package $package$;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class GreeterTest {

    @Test
    void greets_by_name() {
        assertEquals("Hello, World!", new Greeter("world").greet());
        assertEquals("Hello!", new Greeter("").greet());
    }
}

File: kotlin/none/library.g8/src/main/g8/src/test/kotlin/$package$/GreeterTest.kt
package $package$

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class GreeterTest {

    @Test
    fun greets_by_name() {
        assertEquals("Hello, World!", Greeter("world").greet())
        assertEquals("Hello!", Greeter("").greet())
    }
}
